### PR TITLE
Change examples to use genezio latest features.

### PR DESCRIPTION
**Pull request by vladiulianbogdan**

### CODE REVIEW

File: javascript/blockchain/server/genezio.yaml [hunk 1]
```diff
@@ -1,17 +1,2 @@
-name: genezio-blockchain
-region: eu-central-1
-sdk:
-  language: js
-  path: ../client/src/sdk/
-frontend:
-  path: ../client/build/
 scripts:
```
Comment (radu1122) [inline javascript/blockchain/server/genezio.yaml:1]: Shouldn't be one tab on the left?

File: javascript/cron/genezio.yaml [hunk 2]
```diff
@@ -2,13 +2,3 @@
 # Add the paths to classes that you want to deploy in "classes".
 
 name: cron-example
```
Comment (radu1122) [inline javascript/cron/genezio.yaml:4]: Same here

**Review by radu1122 — APPROVED** [review]

**Review by andreia-oca — CHANGES_REQUESTED** [review]
I have a few overall comments.

Two of the projects are still outdated `javascript/hello-world` and `javascript/todo-list-sql`.

The READMEs are outdated.

There is an old sdk directory that is leftover in the codebase:
```
./javascript/todo-list-vue/client/src/sdk
```

And some build directories that should also be cleaned:
```
./javascript/todo-list-vue/client/build
./typescript/webhook/client/build
```

File: javascript/todo-list-sql/client/genezio.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: todo-list
+region: us-east-1
+sdk:
+  language: js
+  path: ../client/src/sdk
+frontend:
+  path: ../client/build
```
Comment (andreia-oca) [inline javascript/todo-list-sql/client/genezio.yaml:7]:
```suggestion
  path: ./build
```

File: typescript/getting-started/client/genezio.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+scripts:
+  preFrontendDeploy: npm run install-prod-sdk && npm install  && npm run build
+frontend:
+  path: ../client/build
```
Comment (andreia-oca) [inline typescript/getting-started/client/genezio.yaml:4]:
```suggestion
  path: ./build
```

File: typescript/quiz-app-prisma/client/genezio.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+scripts:
+  preFrontendDeploy: npm run install-prod-sdk && npm install && npm run build
+frontend:
+  path: ../client/build
```
Comment (andreia-oca) [inline typescript/quiz-app-prisma/client/genezio.yaml:4]:
```suggestion
  path: ./build
```

**Pull request merged**